### PR TITLE
Adding VIRGO lsc files

### PR DESCRIPTION
**Pull request by efajardo**
Adding the VIRGO voms to OSG

### CODE REVIEW

File: vomses [hunk 1]
```diff
@@ -44,3 +44,4 @@
 "dune" "voms2.fnal.gov" "15042" "/DC=org/DC=incommon/C=US/ST=IL/L=Batavia/O=Fermi Research Alliance/OU=Fermilab/CN=voms2.fnal.gov" "dune"
 "supercdms" "voms.slac.stanford.edu" "15001" "/DC=org/DC=incommon/C=US/postalCode=94305/ST=CA/L=Stanford/street=450 Serra Mall/O=Stanford University/OU=SLAC/CN=voms.slac.stanford.edu" "supercdms"
 "dteam" "voms2.hellasgrid.gr" "15004" "/C=GR/O=HellasGrid/OU=hellasgrid.gr/CN=voms2.hellasgrid.gr" "dteam"
+"virgo" "voms.cnaf.infn.it" "15009" "/C=IT/O=INFN/OU=Host/L=CNAF/CN=voms.cnaf.infn.it" "virgo"
```
Comment (edquist) [inline vomses]: Please include a trailing newline at EOF for vomses

Comment (efajardo) [reply vomses]: Fixed

**Comment by efajardo** [comment]
@edquist Can you check back again Carl and merge?

**Comment by efajardo** [comment]
@edquist do we need a JIRA ticket for this?

**Comment by edquist** [comment]
Yes, generally we make a new JIRA ticket like:
https://opensciencegrid.atlassian.net/browse/SOFTWARE-3672

I'm not sure whether any other vo-client updates are scheduled

**Comment by efajardo** [comment]
ah ok I created a new one

https://opensciencegrid.atlassian.net/browse/SOFTWARE-3683

**Pull request merged**